Podfile.lock

```diff
@@ -668,9 +668,6 @@ CHECKOUT OPTIONS:
     :commit: bd76109049d2eeead78478d20d66ffd7b1567dd3
     :git: http://github.com/wordpress-mobile/gutenberg-mobile/
     :submodules: true
-  RNCMaskedView:
-    :commit: d4ccf2bba163679c4550ce6ba0119604cd5e6379
-    :git: https://github.com/react-native-community/react-native-masked-view.git
   RNTAztecView:
```

Comment:
CI is failing because of this. 

`RNCMaskedView` is in the `Podfile`, but this changes removes it from `Podfile.lock`, so after a `pod install` this file gets changed.

Reply:
Thanks @loremattei! 

It took a few tries to repro this locally – I ended up having to delete the whole `Pods` directory to get it working.

Should be fixed now